apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/source/StandardGrpcRdaSource.java

```diff
@@ -124,29 +129,27 @@ public boolean performSmokeTest(RdaSink<TMessage, TClaim> sink) throws Exception
         sink.readMaxExistingSequenceNumber().orElse(MIN_SEQUENCE_NUM);
     log.info("smoke test: startingSequenceNumber={}", startingSequenceNumber);
 
-    // Call the RDA API version service to confirm the API is accessible.
-    final String apiVersion = caller.callVersionService(channel, callOptionsFactory.get());
-    log.info("smoke test: apiVersion={}", apiVersion);
+    if (serverType == RdaSourceConfig.ServerType.Remote) {
```

Comment:
It might be helpful to future devs to spell out, with a comment, why we are skipping non Remote connections.  I know it seems obvious to us, but I doubt thats true for the other BFD engineers

Reply:
Good point.  I have added explanatory text to the javadoc.